src/Umbraco.Web/Services/IconService.cs

```diff
@@ -12,31 +13,24 @@ namespace Umbraco.Web.Services
     public class IconService : IIconService
     {
         private readonly IGlobalSettings _globalSettings;
+        private readonly IHtmlSanitizer _htmlSanitizer;
 
-        public IconService(IGlobalSettings globalSettings)
+        public IconService(IGlobalSettings globalSettings,IHtmlSanitizer htmlSanitizer)
```

Comment:
public IconService(IGlobalSettings globalSettings,IHtmlSanitizer htmlSanitizer)
```suggestion
        public IconService(IGlobalSettings globalSettings, IHtmlSanitizer htmlSanitizer)
```

Reply:
I know it's nitpicking! But it hurts my eyes.. ;-)